spec/integ/crypto/verification.spec.ts

```diff
@@ -1221,6 +1519,52 @@ function expectSendToDeviceMessage(msgtype: string): Promise<{ messages: any }>
     });
 }
 
+/**
+ * Utility to add all needed mocks for secret requesting (to-device of type `m.secret.request`).
+ *
+ * The following secrets are mocked: `m.cross_signing.master`, `m.cross_signing.self_signing`,
+ * `m.cross_signing.user_signing`, `m.megolm_backup.v1`.
+ *
+ *  @returns a map of secret name to promise that will resolve (with the id of the secret request) when the secret is requested.
+ */
+function mockSecretRequestAndGetPromises(): Map<string, Promise<string>> {
+    const mskRequestDefer = defer<string>();
+    const sskRequestDefer = defer<string>();
+    const uskRequestDefer = defer<string>();
+    const backupKeyRequestDefer = defer<string>();
+
+    fetchMock.put(
+        new RegExp(`/_matrix/client/(r0|v3)/sendToDevice/m.secret.request`),
+        (url: string, opts: RequestInit): MockResponse => {
+            const messages = JSON.parse(opts.body as string).messages[TEST_USER_ID];
+            // rust crypto broadcasts to all devices, old crypto to a specific device, take the first one
+            const content = Object.values(messages)[0] as any; //messages[TEST_DEVICE_ID] ? messages[TEST_DEVICE_ID] : messages['*'];
```

Comment:
I don't really understand what this comment means. Is it important? Can you rephrase or delete?

Reply:
So it's too much comment?

I mean it's important if you want to use this for legacy test. Just explains that with rust crypto the secret request is broadcasted to "*" and in legacy just sent to `TEST_DEVICE_ID`

Reply:
It's specifically the `//messages[TEST_DEVICE_ID] ? messages[TEST_DEVICE_ID] : messages['*'];` that I'm complaining about. I suppose it makes sense now that I've thought about it again, but I just don't see that it's adding any value.

Reply:
```suggestion
            const content = Object.values(messages)[0] as any;
```